lib/modules/accountSwitcher.js

```diff
@@ -434,12 +456,42 @@ function _notifySwitchedAccountElsewhere() {
 		message,
 	});
 }
-
+function autoSwitcher(thing) {
+	if(thing.getAuthor() != loggedInUser() && window.location.search.indexOf('accountSwitcher=auto') == -1 ) {
```

Comment:
You should sanity check the type of thing and its relevant option at the very top of this function.

function autoSwitcher(thing) {
     if (thing.isPost()) { if (!module.options.autoSwitchOnPost.value) { return; } }
     if (thing.isComment()) { if (!module.options.autoSwitchOnComment.value) { return; } }